### PR TITLE
Added details for IPS Anywhere for Windows install/uninstall

### PR DESCRIPTION
**Pull request by chadmiddleton**

### CODE REVIEW

**Comment by chadmiddleton** [comment]
Is there anything going on with this pull request?

**Comment by rpdubose** [comment]
@chadmiddleton we'll deploy today or tomorrow.

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (srveit) [inline .idea/compiler.xml]: What is this file used for?

Comment (chadmiddleton) [inline .idea/compiler.xml]: you can leave out all of the .idea updates, I think they got put in on accident

Comment (srveit) [inline .idea/compiler.xml]: @chadmiddleton You'll have to remove them from your branch and then sync again.

**Pull request closed without merging**